### PR TITLE
new components and notifcations

### PR DESCRIPTION
**Pull request by quebonafide**

### CODE REVIEW

File: client/components/quickSearch/quickSearch.module.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+.inputStyles {
+  background-color: $purple;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #949494;
+  color: $white;
+}
+
+.inputStyles:focus {
```
[the remark this answers is not in the document]
Comment (quebonafide) [reply client/components/quickSearch/quickSearch.module.scss:27]: corrected for all

File: client/components/filtr/filter.module.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+@import '../../theme/font';
+@import '../../theme/color';
 
 .searchBackground {
   background-color: $purple-light;
 }
 
 .grayBackground {
-  background-image: url("https://static.wixstatic.com/media/127fea_f5b57c778e3f463e9f88590afb5ed730~mv2.jpg/v1/fit/w_924,h_520/127fea_f5b57c778e3f463e9f88590afb5ed730~mv2.jpg");
+  background-image: url('https://static.wixstatic.com/media/127fea_f5b57c778e3f463e9f88590afb5ed730~mv2.jpg/v1/fit/w_924,h_520/127fea_f5b57c778e3f463e9f88590afb5ed730~mv2.jpg');
```
Comment (jakpolak) [inline client/components/filtr/filter.module.scss:9]: The background image shouldn't be a reference to the Wix site. It is used like this multiple times so please change it in all places.

File: client/pages/courses/addCourse.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ import Notify from '../../components/notifaction'
 import { ToastContainer } from 'react-toastify'
 import 'react-toastify/dist/ReactToastify.css'
 import { NotoficationType } from '../../components/notifaction/types'
```
Comment (konopek94) [inline client/pages/courses/addCourse.tsx:14]: typo ;p

Comment (konopek94) [reply client/pages/courses/addCourse.tsx:14]: `NotoficationType`

File: client/components/quickSearch/index.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            </button>
+          </div>
+        </form>
+        {customLink && customLink}
+        <div className="row no-gutters justify-content-start align-items-center">
+          <div className="col-auto">
+            <p className={`${style.quickSearch} mb-0`}>Szybkie filtry:</p>
+          </div>
+          <div className="row ml-2">
+            {quickSearch.map((ele: { src: string; alt: string }, index: number) => {
```
Comment (konopek94) [inline client/components/quickSearch/index.tsx:62]: Could u move this inline typing `{ src: string; alt: string }` to separate type/interface?

File: client/variables/variables.ts [hunk 5]
```diff
@@ -0,0 +1,4 @@
+// notification timers
+export const shortTimer = 2000
+export const mediumTimer = 3500
+export const longTimer = 5000
```
Comment (jakpolak) [inline client/variables/variables.ts:4]: I would move these variables to the Notification file. They are closely related to notifications and it feels like overkill to do it in a separate file.

**Review by jakpolak — CHANGES_REQUESTED** [review]

File: client/components/filtr/filter.module.scss (hunk 2 above)
Comment (quebonafide) [reply client/components/filtr/filter.module.scss:9]: also done for add course form

**Review by jakpolak — APPROVED** [review]

**Review by konopek94 — APPROVED** [review]

File: client/components/quickSearch/index.tsx [hunk 6]
```diff
@@ -59,7 +64,7 @@ export const QuickSearch: FC<IProps> = ({ title, customLink }: IProps): ReactEle
             <p className={`${style.quickSearch} mb-0`}>Szybkie filtry:</p>
           </div>
           <div className="row ml-2">
-            {quickSearch.map((ele: { src: string; alt: string }, index: number) => {
+            {quickSearch.map((ele: Element) => {
```
Comment (konopek94) [inline client/components/quickSearch/index.tsx:67]: better to call it as `imageElement` or sm

**Review by konopek94 — APPROVED** [review]

**Pull request merged**